test/plugin/test_in_tail.rb

```diff
@@ -3265,6 +3265,7 @@ def test_refreshTW_during_rotation
       inode_0 = tail_watchers[0]&.ino
       inode_1 = tail_watchers[1]&.ino
       inode_2 = tail_watchers[2]&.ino
```

Comment:
Looks like we don't need to assert these inode values in this test.
Could you please remove them in this PR?

Reply:
OK, I will remove them.
Thanks

Reply:
Fixed with https://github.com/fluent/fluentd/pull/4710/commits/8880cb679aa131c0692d43fe3bab232ce812cf06